pages/sessions.jsx

```diff
@@ -0,0 +1,215 @@
+      "<p>Turn off your screen. Close your book. End that meeting. Pick up a sketchpad, a pair of scissors, a hot-glue gun, some parcel tape and come cry “If We Build It They Will Come.” This is a springboard for tomorrow and welcoming place for those inclusive citizens and communities.</p>",
+    wranglers: [
+      {
+        name: "Jon Rogers"
+      },
+      {
+        name: "Ian Forrester"
+      },
+      {
+        name: "Jasmine Cox"
+      }
+    ]
+  },
+  {
+    name: "Participation",
+    description:
+      "<p>Participation is at the heart of Mozilla’s approach to learning. But participation doesn’t just happen — it’s built through great design and great leadership.</p>" +
+      "<p>The Participation Space will be a mixture of workshops, hack sessions, open challenges and product tinkering aimed at building participation and developing leaders who can inspire others. Participants will cover personal leadership exploration, skill development and community building.</p>",
+    wranglers: [
+      {
+        name: "George Roter",
+        twitter: "@geroter"
+      },
+      {
+        name: "Emma Irwin",
+      }
+    ]
+  },
+  {
+    name: "Youth Zone",
+    description:
+      "<p>There’s sometimes a fundamental clash between youth and adults on the Web. Young people discover platforms and activities that are important and meaningful to them — but adults disagree, wishing youngsters would invest their time elsewhere. In the Youth Zone Space, we’ll explore and aim to reconcile this conflict.</p>" +
+      "<p>Young people and adults will work in tandem across three workshops and 30 sessions. Together, we’ll explore new technology and how it’s relevant to all participants. We’ll also build things together, and learn about fulfilling careers based in new media.</p>",
+    wranglers: [
+      {
+        name: "Dorine Flies",
+        twitter: "@dorineflies"
+      },
+      {
+        name: "Harry Smith"
+      }
+    ]
+  },
+  {
+    name: "Localization",
+    description:
+      "<p>All languages and cultures are uniquely different and valuable — and they should be treated this way on the Web, too. In order for the Web to empower everyone, it must be adapted to diverse communities and peoples. The next billion people to come online shouldn’t find an English-only Web.</p>" +
+      "<p>In the Localization Space, participants learn language localization skills and nuances, how to include diversity in products, and how to bring this philosophy back to their local communities.</p>",
+    wranglers: [
+      {
+        name: "Heather Bailey"
+      },
+      {
+        name: "Dwayne Bailey"
+      }
+    ]
+  }
+];
+
+var Space = React.createClass({
```

Comment:
I think the reason we are calling this page "sessions" instead of "spaces" is cuz Spaces seems like a vague name to put on the nav bar.  Space is indeed the term we have been using for this year's MozFest.